src/app/pages/admin/search/search.component.ts

```diff
@@ -87,7 +87,7 @@ export class SearchComponent {
       .pipe(map(([user, position]) => {
         if (position === null) return null;
         const opt: MarkerOptions = {
-          title: `${user.firstName} ${user.surname}`,
+          title: `${user.firstName} ${user.prefix} ${user.surname}`,
```

Comment:
Prefix is optional, so it could be empty. I guess we can live with the double space (if that is even visible in the app) but has this been tested for other side-effects?

Reply:
Suggestion: If prefix is indeed optional then you can use 

`${user.firstName}${user.prefix ? ` ${user.prefix}` : ''} ${user.surname}`

```
const user = {
  firstName: 'Jane',
  prefix: '',
  surname: 'Doe'
}

```
returns: 'Jane Doe'

```
const user = {
  firstName: 'Jane',
  prefix: 'van',
  surname: 'Doe'
}

```
returns: 'Jane van Doe'

Reply:
Thanks for the suggestion